transaction-view/benches/bytes.rs

```diff
@@ -0,0 +1,90 @@
+use {
+    agave_transaction_view::bytes::{optimized_read_compressed_u16, read_compressed_u16},
+    bincode::{serialize_into, DefaultOptions, Options},
+    criterion::{black_box, criterion_group, criterion_main, Criterion, Throughput},
+    solana_sdk::{
```

Comment:
if you copy the definition of PACKET_DATA_SIZE and use the `solana_short_vec`  crate you can avoid depending on solana-sdk

Reply:
This is just the initial parsing helpers - the tx type will eventually be exposing `Signature` which is `sdk` 😢

Reply:
When #2054 gets merged `Signature` will also be available outside the SDK

Reply:
Just curious if there are actions to this comment? Could remove SDK, then race to see if #2054 got merged first or tx type starts to expose Signature.

Reply:
think its simpler / more straight-forward to keep interface as is for now.
Once the other PR gets merged we can simplify the dependencies, and the interface itself doesn't change.